sormas-api/src/main/resources/strings.properties

```diff
@@ -995,6 +996,9 @@ DiseaseNetworkDiagram.lowRisk = Low risk
 DiseaseNetworkDiagram.selectByClassification = Select by Classification
 DiseaseNetworkDiagram.subheading = The arrows indicate the direction of transmission
 
+# Unlink events
+confirmationUnlinkCaseFromEvent = Are you sure you want to unlink this case from this event ?
```

Comment:
Move the property to the section in this file that contains the other confirmations and remove the whitespace before the question mark